### PR TITLE
make consistent with the sim doc

### PR DESCRIPTION
**Pull request by maqi**

### CODE REVIEW

File: src/network.rs [hunk 1]
```diff
@@ -168,9 +172,15 @@ impl Network {
                 Response::Reject(_) => {
                     stats.rejections += 1;
                 }
-                Response::Relocate(node) => {
-                    stats.relocations += 1;
-                    self.handle_relocate(node)
+                Response::RelocateRequest(src, name, node) => {
```
Comment (madadam) [inline src/network.rs:175]: I think you can remove `Response::RelocateRequest` variant and this branch and use `Response::Send(Request::RelocateRequest(...))` instead.

File: src/section.rs [hunk 2]
```diff
@@ -105,47 +128,49 @@ impl Section {
...
-                responses
-            } else if node.is_adult(params) {
-                self.try_relocate(params, None)
-            } else {
-                Vec::new()
+            let mut responses = self.update_elders(params, true);
+            responses.extend(self.try_merge(params));
+            if node.is_adult(params) {
+                if let Some(last_live) = self.chain.last_live() {
+                    responses.extend(self.try_relocate(params, last_live));
```
Comment (madadam) [inline src/section.rs:170]: If merge has been triggered, should we still trigger relocation?

File: src/network.rs (hunk 1 above)
Comment (maqi) [reply src/network.rs:175]: I don't think so. At that moment, from the source section perspective, it doesn't know which section to relocate node to, hence can't fill the prefix field for Response::Send.
It has to be the network to figure out which section can accept it.

File: src/section.rs (hunk 2 above)
Comment (maqi) [reply src/section.rs:170]: if we are the section triggering merge, in `try_relocate`, the number of nodes will be checked as well, and there will be no relocation happen.
if we are the section being told to merge, our nodes is already empty, so relocation won't happen either.

so, no need to change code?

File: src/network.rs (hunk 1 above)
Comment (madadam) [reply src/network.rs:175]: :+1:

File: src/section.rs (hunk 2 above)
Comment (madadam) [reply src/section.rs:170]: :+1:

**Pull request merged**